mapie/regression/quantile_regression.py

```diff
@@ -609,8 +613,12 @@ def fit(
                 else:
                     cloned_estimator_.set_params(**params)
                 self.estimators_.append(fit_estimator(
-                    cloned_estimator_, X_train, y_train, sample_weight_train
-                ))
+                    cloned_estimator_,
+                    X_train,
+                    y_train,
+                    sample_weight_train,
+                    **fit_params,
+                    ))
                 y_calib_preds[i] = self.estimators_[-1].predict(X_calib)
```

Comment:
Just a writing style suggestion:
```suggestion
                ))
```